### DISCUSSION TITLE
Passwd , never requested

### DISCUSSION THREAD

**Discussion by badwolf8u2** [community/community#114822] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

with mingw64:/c < I cant find a passwd to open these type of files  / also Git Bash /or Docker desktop / or Alpine ? all use the same passwd " But I never have added any password  or passwd's

**Comment by Akash1134** [comment]
Hi @badwolf8u2 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by Yaseenkhan09** [reply]
I open facebook id i have only id number
